Content.Server/Temperature/Systems/EntityHeaterSystem.cs

```diff
@@ -57,7 +58,8 @@ private void OnExamined(EntityUid uid, EntityHeaterComponent comp, ExaminedEvent
 
     private void OnGetVerbs(EntityUid uid, EntityHeaterComponent comp, GetVerbsEvent<AlternativeVerb> args)
     {
-        if (!args.CanAccess || !args.CanInteract)
+        // cant turn the knob if you have no hands!
+        if (!args.CanAccess || !args.CanInteract || !HasComp<HandsComponent>(args.User))
```

Comment:
This would still let someone with a hands components but 0 hands get the verb.

Reply:
is there a helper for this that other systems use

Reply:
`CanInteract` is set via `ActionBlockerSystem`'s `CanInteract` whcih raises `InteractionAttemptEvent`. So perhaps the BodySystem needs to subscribe to `InteractionAttemptEvent` and check if the user has hands. Since this hand interaction is probably checked in a lot of places, you'd have to check very carefully if there are unintended consequences, e.g. not being able to sit down on a chair if you don't have hands.